lib/internal/worker.js

```diff
@@ -47,6 +47,15 @@ const kIncrementsPortRef = Symbol('kIncrementsPortRef');
 
 const debug = util.debuglog('worker');
 
+const messageTypes = {
+  UP_AND_RUNNING: 'kUpAndRunning',
+  COULD_NOT_SERIALIZ_EERROR: 'kCouldNotSerializeError',
```

Comment:
I think the `_` ended up in the wrong place? ;)

Reply:
Oops, my fault /w\